lib/standalone.mjs

```diff
@@ -111,7 +111,7 @@ export default async function standalone({
 		emitDeprecationWarning(
 			'"github" formatter is deprecated.',
 			'GITHUB_FORMATTER',
-			'Please use "stylelint-actions-formatters" instead.',
+			'See https://github.com/stylelint/awesome-stylelint/blob/main/README.md#formatters for alternative formatters.',
```

Comment:
```suggestion
			'See https://stylelint.io/awesome-stylelint#formatters for alternative formatters.',
```

Reply:
Applied in this commit.
[Replace url](https://github.com/stylelint/stylelint/pull/8115/commits/eb758e3d66bf42b4e3908171b7fb3622e2a902ef)